powershell/enterprisePolicies/README.md

```diff
@@ -0,0 +1,150 @@
+# Power Platform Enterprise Policies PowerShell Scripts
+
+These scripts introduces the capabilities of the Enterprise Policies ARM resources. The sample scripts provides capabilities to create, update, delete and get enterprise policies.  
```

Comment:
I suggest replacing with this text: "These scripts automate managing (create, update, get, delete) Power Platform Enterprise Policies as Azure resources. In addition, we are providing sample scripts on how to associate these policies with Power Platform environments.